devtools/jenkins/test_examples.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+envname=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1`
+sed -i -- 's/tensorflow$/tensorflow-gpu/g' scripts/install_deepchem_conda.sh
+export python_version=2.7
+bash scripts/install_deepchem_conda.sh $envname
+source activate $envname
+python setup.py install
+conda install -y jupyter
+conda install -y nbconvert
```

Comment:
I don't think you need these jupyter specific imports

but on a side note do we want to make jupyter etc part of the default install?